Author: Pino Toscano <pino@kde.org>
Description: Fixes FTBFS on GNU/kFreeBSD by creating new Q_OS_FREEBSD_KERNEL.
Last-Update: 2010-10-06

---
 src/corelib/global/qglobal.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/src/corelib/global/qglobal.h
+++ b/src/corelib/global/qglobal.h
@@ -226,6 +226,8 @@ namespace QT_NAMESPACE {}
 #elif defined(__FreeBSD__) || defined(__DragonFly__)
 #  define Q_OS_FREEBSD
 #  define Q_OS_BSD4
+#elif defined(__FreeBSD_kernel__)
+#  define Q_OS_FREEBSD_KERNEL
 #elif defined(__NetBSD__)
 #  define Q_OS_NETBSD
 #  define Q_OS_BSD4
